trestle/common/model_utils.py

```diff
@@ -36,6 +37,9 @@
 
 logger = logging.getLogger(__name__)
 
+# Generic type var
+TG = TypeVar('TG')
```

Comment:
I think we have TG variable in `common_types` shall we use that one instead?

Reply:
Thanks - it wasn't needed anyway so good catch.  removed.